rhel/rshim.spec.in

```diff
@@ -36,6 +33,16 @@ via the virtual console or network interface.
 %install
 %make_install
 
+%pre
+if ! modprobe cuse; then
+  echo "Error: cuse.ko module is not available. Do one of the following:"
+  echo "  1. Build and install cuse.ko module."
+  echo "  2. Install a package that has cuse.ko, like kernel-modules-extra."
```

Comment:
Change kernel-modules-extra to kernel-modules-extra-$(uname -r) to be more accurate.